packages/shared/src/components/PostOptionsMenu.tsx

```diff
@@ -208,6 +208,21 @@ export default function PostOptionsMenu({
     );
   };
 
+  const onUnblockSource = async (): Promise<void> => {
+    const { successful } = await onFollowSource({
+      source: post?.source,
+      requireLogin: true,
+    });
+
+    if (!successful) {
+      return;
```

Comment:
Should we display an error to the user if it does not succeed?

Reply:
Not succeeding in this case can also be just user not logged in. Also follows the same logic as block so don't want to complicate it.